common/test/acceptance/pages/lms/courseware.py

```diff
@@ -209,13 +209,13 @@ def has_submitted_exam_message(self):
         """
         return self.q(css="div.proctored-exam.completed").visible
 
-    def content_hidden_past_due_date(self, content_type="subsection"):
+    def content_hidden_past_due_date(self):
         """
         Returns whether the "the due date for this ___ has passed" message is present.
         ___ is the type of the hidden content, and defaults to subsection.
         This being true implies "the ___ contents are hidden because their due date has passed".
         """
-        message = "The due date for this {0} has passed.".format(content_type)
+        message = "this assignment is no longer available"
```

Comment:
Not here - the dialog we're checking for here can vary in how it begins, so I pulled out this fragment.

<img width="675" alt="screen shot 2017-01-05 at 10 31 19 am" src="https://cloud.githubusercontent.com/assets/7373924/21686136/49f76e8a-d332-11e6-92fc-5157690be67f.png">